.github/dependabot.yml

```diff
@@ -1,6 +1,7 @@
 version: 2
 updates:
-  - package-ecosystem: "pub"
+  - package-ecosystem: pub
+    versioning-strategy: increase
```

Comment:
A bit of an unrelated change, but dependabot's default setting is `widen` for pub

This basically means: "probably dont change the pubspec.yaml file"

We dont want this, always change the pubspec.yaml file to enforce that latest versions of pubspec files

(background issue: https://github.com/dependabot/dependabot-core/issues/4979)